mmv1/third_party/terraform/resources/resource_sql_user.go

```diff
@@ -83,6 +83,28 @@ func resourceSqlUser() *schema.Resource {
                 The default is the database's built-in user type. Flags include "BUILT_IN", "CLOUD_IAM_USER", or "CLOUD_IAM_SERVICE_ACCOUNT".`,
 				ValidateFunc: validation.StringInSlice([]string{"BUILT_IN", "CLOUD_IAM_USER", "CLOUD_IAM_SERVICE_ACCOUNT", ""}, false),
 			},
+			"sql_server_user_details": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				ForceNew: true,
```

Comment:
```suggestion
```

Given neither subfield is `ForceNew`, it's unlikely we want the parent to be `ForceNew` either.

Reply:
removed